gbfs.md

```diff
@@ -190,22 +229,90 @@ stations          | Yes       | Array that contains one object per station in th
 \- region_id       | Optional  | ID of the region where station is located (see [system_regions.json](#system_regionsjson))
 \- post_code       | Optional  | Postal code where station is located
 \- rental_methods  | Optional  | Array of enumerables containing the payment methods accepted at this station. <br />Current valid values (in CAPS) are:<br /><ul><li>KEY _(i.e. operator issued bike key / fob / card)_</li> <li>CREDITCARD</li> <li>PAYPASS</li> <li>APPLEPAY</li> <li>ANDROIDPAY</li> <li>TRANSITCARD</li> <li>ACCOUNTNUMBER</li> <li>PHONE</li> </ul> This list is intended to be as comprehensive at the time of publication as possible but is subject to change, as defined in [File Requirements](#file-requirements) above
-\- capacity        | Optional  | Number of total docking points installed at this station, both available and unavailable
+\- capacity        | Optional  | Number of total docking points installed at this station, both available and unavailable, regardless of what vehicle types are allowed at each dock. This field should only be used if every dock at the station is able to accept every vehicle type used in the system.
+\- vehicle_type_capacity        | Optional  | An object where each key is a vehicle_type_id as described in [vehicle_types.json](#vehicle_typesjson) and the value is a number representing the total docking points installed at this station, both available and unavailable for the specified vehicle type.
+
+Example:
+
+```json
+{
+  "last_updated": 1434054678,
+  "ttl": 0,
+  "data": {
+    "stations": [
+      {
+        "station_id": "pga",
+        "name": "Parking garage A",
+        "lat": 12.34,
+        "lon": 45.67,
+        "vehicle_type_capacity": {
+          "abc123": 7,
+          "def456": 9
+        }
+      }
+    ]
+  }
+}
+```
 
 ### station_status.json
 
 Field Name            | Required  | Defines
 --------------------- | ----------| ----------
 stations              | Yes       | Array that contains one object per station in the system as defined below
 \- station_id          | Yes       | Unique identifier of a station (see station_information.json)
-\- num_bikes_available | Yes       | Number of bikes available for rental
+\- num_bikes_available | Conditionally required       | Number of vehicles of all types available for rental. This field is not required if the `vehicles_available` field has been defined.
```

Comment:
Thoughts on deprecating this since it is bike specific? Not sure if making breaking changes are out of scope here, since versioning is still up for discussion.

Reply:
I do like the idea of versioning and would change this and rename everything that says "bike" to "vehicle", but will await guidance from the community.